javascript/ch_08/Q_2.js

```diff
@@ -0,0 +1,34 @@
+const box = {
+  locked: true,
+  unlock() { this.locked = false; },
+  lock() { this.locked = true;  },
+  _content: [],
+  get content() {
+    if (this.locked) throw new Error("Locked!");
+    return this._content;
+  }
+};
+function withBoxUnlocked(body) {
+  let status = box.locked;
```

Comment:
unless a variable (like status) is used at multiple places, you could have simply written this:

`if (!box.locked)`